### ISSUE TITLE
Possible bug in admin generator for code/autogen/coldefs.h

### ISSUE THREAD

**Issue by Daedalus11069** [rezalas/riftshadow#79]
During creation of the Utilities controller, I came upon a problem; the `bans` table in `rift_core` was not created during the original admin interface, meaning it wasn't accounted for in the generation portion of that interface for generating the `code/autogen/coldefs.h` file.

The specific problem plaguing that generation script is that there was no cast type for the `datetime` field type.

So I implemented a slight hack on line 378 of the `genColDefs()` method of the Utilities controller:

```php
} elseif (
	$f_type === "string" ||
	$f_type === "blob" ||
	$f_type === 'datetime' // FIXME: Fix this hack please
) {
	$m_var .= "tvar{$pchar}{$f_name}.NoDeallocSetTo(row[{$d_name}]);";
}
```

In short, I cast the `datetime` field to a string.  I don't know if this will have problems later down the line, so here's the bug report to keep track of it.
